### PR TITLE
feat(GRA-609aq): add basic auth and sso connect and uninstall

### PR DESCRIPTION
**Pull request by Aceix**

### CODE REVIEW

File: gui/exported.go [hunk 1]
```diff
@@ -88,3 +90,61 @@ func (app *App) GoLeaveNetwork(networkName string) (any, error) {
...
+	err := functions.Join(flags)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+
+	return nil, nil
+}
+
+// App.GoJoinNetworkBySso joins a network by SSO
```
Comment (mattkasun) [inline gui/exported.go:120]: comment needs updating copy/paste errror?

**Review by mattkasun — CHANGES_REQUESTED** [review]
delete the go.mod/go.sum from the gui dir

File: gui/exported.go (hunk 1 above)
Comment (0xdcarns) [reply gui/exported.go:120]: perhaps we need to consolidate after this merge to ensure it works..

Comment (0xdcarns) [reply gui/exported.go:120]: agreed

**Comment by 0xdcarns** [comment]
> delete the go.mod/go.sum from the gui dir

perhaps we need to consolidate after this merge to ensure it works..

File: gui/exported.go (hunk 1 above)
Comment (Aceix) [reply gui/exported.go]: resolved

**Review by mattkasun — APPROVED** [review]

**Pull request merged**